pkg/proxy/ipvs/README.md

```diff
@@ -338,10 +338,23 @@ Kube-proxy will run in iptables mode by default in a cluster deployed by [kubead
 
 If you are using kubeadm with a [configuration file](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file), you can specify the ipvs mode adding `SupportIPVSProxyMode: true` below the `kubeProxy` field.
 
+1. Kubernetes before v1.10
 ```json
 kind: MasterConfiguration
 apiVersion: kubeadm.k8s.io/v1alpha1
 ...
+kubeProxy:
+  config:
+    featureGates:
+      SupportIPVSProxyMode: true
+    mode: ipvs
+...
+```
+2. Kubernetes v1.11 and later
+```json
+kind: MasterConfiguration
+apiVersion: kubeadm.k8s.io/v1alpha2
+...
```

Comment:
just remove both kubeadm config examples in this document and do this instead:

----------


If you are using kubeadm with a [configuration file](https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-init/#config-file), you have to add `mode: ipvs` and also add `SupportIPVSProxyMode: true` below the `kubeProxy` field as part of the kubeadm configuration.
```
kubeProxy:
  config:
    featureGates:
      SupportIPVSProxyMode: true
    mode: ipvs
```
Note that in Kubernetes 1.11 and later, `SupportIPVSProxyMode` is set to `true` by default.

----------

Reply:
@neolit123   Thanks a lot.